.github/workflows/e2e.yaml

```diff
@@ -138,15 +151,100 @@ jobs:
           prometheus_region: ${{ vars.PROMETHEUS_REGION }}
           enable_local_zones: ${{ inputs.suite == 'LocalZone' }}
           cleanup: ${{ inputs.cleanup }}
+          cidr_block: ${{ inputs.cidr_block }}
+          codebuild_sg: ${{ vars[format('{0}_CODEBUILD_SG', env.CODEBUILD_REGION)] }}
+          codebuild_vpc: ${{ vars[format('{0}_CODEBUILD_VPC', env.CODEBUILD_REGION)] }}
+          codebuild_role: ${{ vars[format('{0}_CODEBUILD_ROLE', env.CODEBUILD_REGION)] }}
+      - name: run tests for private cluster
```

Comment:
Can we move this into its own GitHub action? So that e2e.yaml does not feel cramped